runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc

```diff
@@ -220,7 +220,7 @@ void FullyConnectedLayer::prepare()
     }
   }
 
-#if defined(__ARM_NEON__) && defined(USE_RUY_GEMV)
+#if defined(__ARM_NEON) && defined(USE_RUY_GEMV)
```

Comment:
Could you change this like below?
```suggestion
#if (defined(__ARM_NEON__) || defined(__ARM_NEON)) && defined(USE_RUY_GEMV)
```

Reply:
@ragmani Thanks! I've applied your suggestion.